spec/github_changelog_generator_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+describe GitHubChangelogGenerator::ChangelogGenerator do
```

Comment:
In this project, prefer the non-monkey-patched form:

```suggestion
RSpec.describe GitHubChangelogGenerator::ChangelogGenerator do
```

Perhaps we can enable config.disable_monkey_patching! for RSpec.